src/content/templates/template books/Special/Special.less

```diff
@@ -2,3 +2,12 @@
 // Also, we import the css from the new style quiz so that any legacy quiz pages will look right,
 // since they will be expecting to find their css here.
 @import "../Activity/simpleComprehensionQuiz.less";
+
```

Comment:
I assume all this will change decides how the background color etc. will work. But for now at least making things non-white allows us to see the current white outline showing where the grid is